app/features/index.py

```diff
@@ -0,0 +1,51 @@
+'''
```

Comment:
This shouldn't be in a whole new directory. Please use the current structure of internal/routers